contrib/update-my-copyright.pl

```diff
@@ -67,6 +67,9 @@
 my $my_search_name = "Cisco";
 my $my_formal_name = "Cisco Systems, Inc.  All rights reserved.";
 
+# Protected directories
+my @protected = qw(pmi2x/pmix/ hwloc1113/hwloc/ libevent2022/libevent/);
```

Comment:
sure - I'm just not enough of a perl expert to know how to do it. I'll happily take your diff.